samples/dotnet_core_managed_gateway/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using Microsoft.Azure.Devices.Gateway;
+
+namespace dotnet_core_managed_gateway
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                if (args.Length < 1 || File.Exists(args[0]) == false)
```

Comment:
Yes `args.Length` returns a signed int (because...VBScript), but the language guarantees it will never be negative. So I would test `args.Length == 0` to be clear.
Also for C# readers, `!File.Exists(args[0])` is more familiar than `== false`.